src/main/java/seedu/bookmark/model/Model.java

```diff
@@ -14,6 +14,7 @@
 public interface Model {
     /** {@code Predicate} that always evaluate to true */
     Predicate<Book> PREDICATE_SHOW_ALL_BOOKS = unused -> true;
+    int MAX_BOOK_CAPACITY = 100;
```

Comment:
Maybe this variable should be public static final?

Reply:
variables in interfaces are already public static final by default! :-)

Reply:
Oooh okay didn't know that :D